### DISCUSSION TITLE
aarch64 support - for Neoverse IP ( N1, V2 = Nvidia Grace )

### DISCUSSION THREAD

**Discussion by allaukik** [NVlabs/sionna#401] [Q&A]
Hi, 

I was able to install and run Sionna on M1 macOs. But when I try installing/building it for another aarch64 platform - arm neoverse N1 - it does not build for me. Has anyone tried building it for - say Nvidia Grace ( an Arm Neoverse V2) platform?

**Comment by SebastianCa** [comment]
This seems related to issue #423, where a more comprehensive discussion is provided.

**Reply by merlinND** [reply]
Closing this as duplicate, discussion can continue in #423.
